python/glow/gwas/log_reg.py

```diff
@@ -23,8 +23,8 @@ def logistic_regression(
         phenotype_df: pd.DataFrame,
         covariate_df: pd.DataFrame = pd.DataFrame({}),
         offset_df: pd.DataFrame = pd.DataFrame({}),
-        # TODO: fallback is probably not the best name
-        fallback: str = 'none',  # TODO: Make approx-firth default
+        correction: str = 'none',  # TODO: Make approx-firth default
```

Comment:
nit: I think the term `correction` is misleading for this argument. Something like `alternative` may make more sense.

Reply:
Correction is the term used in the regenie paper to refer to Firth/SPA.